testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -1042,6 +1042,84 @@ def ag(self, request):
         return group
 
     @pytest.fixture()
+    def unordered_ag(self, ag):
```

Comment:
@mnmelo can you check this please? As far as I'm aware all these were changed in #3012?

Reply:
Do you mean the non-seeded RNG use? I agree it should be stabilized.

Reply:
No, I meant more, out of all this code from `master` does any of this need to be kept or was it all intentionally removed in `develop`?